plugin/fzf.vim

```diff
@@ -26,10 +26,62 @@ if exists('g:loaded_fzf')
 endif
 let g:loaded_fzf = 1
 
+function! fzf#shellescape(path)
+  if s:is_win
+    let shellslash = &shellslash
+    try
+      set shellslash
+      return '"'.shellescape(a:path).'"'
+    finally
+      let &shellslash = shellslash
+    endtry
+  endif
+  return shellescape(a:path)
+endfunction
+
+function! fzf#getcwd()
+  if s:is_win
+    let shellslash = &shellslash
+    try
+      set noshellslash
+      return getcwd()
+    finally
+      let &shellslash = shellslash
+    endtry
+  endif
+  return getcwd()
+endfunction
+
+function! fzf#fnamemodify(fname, mods)
+  if s:is_win
+    let shellslash = &shellslash
+    try
+      set noshellslash
+      return fnamemodify(a:fname, a:mods)
+    finally
+      let &shellslash = shellslash
+    endtry
+  endif
+  return fnamemodify(a:fname, a:mods)
+endfunction
+
+function! fzf#expand(fmt)
+  if s:is_win
+    let shellslash = &shellslash
+    try
+      set noshellslash
+      return expand(a:fmt)
+    finally
+      let &shellslash = shellslash
+    endtry
+  endif
+  return expand(a:fmt)
+endfunction
+
```

Comment:
I'll think about it and let you know. I need to find time to test the plugin on Windows before making the decision. Thanks.